src/type/timer.f90

```diff
@@ -45,99 +47,164 @@ module xtb_type_timer
 
 contains
 
+!--------------------------------------------
+!> To initialize timer
+!--------------------------------------------
 subroutine allocate_timer(self,n,verbose)
+   
    implicit none
+   !> dummy arg list
    class(tb_timer),intent(inout) :: self
+      !! instance of timer
```

Comment:
Please be consistent, either add all comments before or after the variable name. In `xtb` I have been using comments before the variables so far.